src/java.base/windows/classes/sun/util/locale/provider/HostLocaleProviderAdapterImpl.java

```diff
@@ -75,7 +75,7 @@
 
     // CalendarData value types
     private static final int CD_FIRSTDAYOFWEEK = 0;
-    private static final int CD_MINIMALDAYSINFIRSTWEEK = 1;
```

Comment:
Do we want to keep the naming consistent, doing the same change to, e.g. the corresponding macosx impl?

Reply:
The constants are for native methods, which differ between macOS and Windows. Thus I thought it would be clearer to align the name with Windows' constants.

Reply:
Make sense.